cmd/cmdutil/helpers_test.go

```diff
@@ -0,0 +1,44 @@
+package cmdutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidateIntOrPercantageValue(t *testing.T) {
+	assert := assert.New(t)
+
+	t.Run("No input is false", func(t *testing.T) {
```

Comment:
Since you're using assert, you can change these tests to use `assert.False(actual)` or `assert.True(actual)` as appropriate. :)